index.js

```diff
@@ -24,37 +25,54 @@ if (argv.help) {
   console.log('--cacheFile=<path>       An absolute path to a JSON cache file for looking up missing licenses.');
   console.log('--ignore=<name1, name2>  Comma separated package names to ignore.');
   console.log('--production             Search for licenses on production dependencies. Default is development.');
+  console.log('--readme                 Use the readme as a fallback when LICENSE is not available.');
   console.log('--help                   This help.');
   process.exit(0);
 }
 
 const production = !!argv.production;
+const readme = !!argv.readme;
 const { directory, cacheFile, ignore } = argv;
 
 const ignoreSet = ignore ? new Set(ignore.split(',')) : new Set();
 const cache = cacheFile ? JSON.parse(fs.readFileSync(cacheFile, 'utf-8')) : null;
 
 console.log(`Searching for ${production ? 'production' : 'development'} licenses in ${directory}`);
 
-nlf.find({ directory, production }, (err, npmPackages) => {
+let promises = [];
+
+lc.init({ start: directory, production }, (err, npmPackages) => {
   if (err) {
     exit(err);
   }
 
   const builders = [new TextBuilder(), new JSONBuilder()];
 
-  npmPackages.forEach(npmPackage => {
-    const { name, version } = npmPackage;
+  _.forEach(npmPackages, (npmPackage, nameVer) => {
+    const [name, version] = [_.initial(nameVer.split('@')).join('@'), _.last(nameVer.split('@'))];
```

Comment:
Whats the need to build the array and then immediately destructure it?

Reply:
No reason really, I can change that. I was actually just converting the pieces and then ended up doing this when testing it etc, forgot to make this less stupid